TraCITestClient output file. Date: Mon Feb 02 15:11:44 2009


-> Command sent: <SimulationStep>:
  TargetTime=11 PosFormat=1
.. Command acknowledged (1), [description: ]
.. Received Response <MoveNode>:
  nodeId=0 targetTime=11 2D-Position: x=33.4375 y=491.75
.. Received Response <MoveNode>:
  nodeId=1 targetTime=11 2D-Position: x=2961.16 y=1008.25
.. Received Response <MoveNode>:
  nodeId=2 targetTime=11 2D-Position: x=2981.54 y=1008.25
.. Received Response <MoveNode>:
  nodeId=3 targetTime=11 2D-Position: x=2991.95 y=1008.25

-> Command sent: <Scenario>:
  flag=0 domain=1 domainId=0 variable=18 valueDataType=0 StringValue=notexisting_vehicle
.. Answered with error to command (115), [description: Vehicle with internal id notexisting_vehicle not existing or not accessible via TraCI]

-> Command sent: <SimulationStep>:
  TargetTime=12 PosFormat=1
----
Error while receiving command: tcpip::Socket::receive() @ recv: No error
----
Error while sending command: send failed: Software caused tcpip::Socket abort
